standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreDirectSql.java

```diff
@@ -1772,23 +1774,29 @@ private long partsFoundForPartitions(
           + " and \"ENGINE\" = ? "
           + " group by \"COLUMN_NAME\", \"COLUMN_TYPE\"";
       start = doTrace ? System.nanoTime() : 0;
-      query = pm.newQuery("javax.jdo.query.SQL", queryText);
-      qResult = executeWithArray(query, prepareParams(catName, dbName, tableName, partNames, colNames, engine),
```

Comment:
I think we should get rid of forward declaring `qResult` with null/etc values - it may aid the escape of a query bound object

forward declaring `queryText` may also doesn't have any benefits

Reply:
Done